service/src/main/java/uk/gov/hmcts/reform/fpl/model/CaseData.java

```diff
@@ -231,4 +232,8 @@ private void setCaseManagementOrder_Judiciary(CaseManagementOrder order) {
         return isEmpty(getRespondents1()) || getRespondents1().size() <= seqNo
             ? empty() : Optional.of(getRespondents1().get(seqNo).getValue());
     }
+
+    public String getFurtherDirectionsText() {
+        return Optional.ofNullable(orderFurtherDirections).map(FurtherDirections::getDirections).orElse("");
+    }
```

Comment:
I think I would prefer something like defaultIfNull(orderFurtherDirections, orderFurtherDirections.builder().build())... thoughts?

Reply:
The mapping captures the case of when `FurtherDirections::getDirections` is null as well so I personally prefer the Optional

Reply:
are directions mandatory? They probably should be if I select yes 🤔

Reply:
and then cool, can leave as is 👍

Reply:
Checked and it wasn't mandatory so made it so, left this as is though as I just want to get it into testing